### ISSUE TITLE
Bug: Doesn't show volume for non-default device

### ISSUE THREAD

**Issue by adx74** [ShankarBUS/ModernFlyouts#778]
**Describe the bug:**

When i slide the know of my Logitech Z407, it's normally showing the volume of the Z407 Audio Device, and not the default Audio Device.

But with ModernFlyouts it's just show the default Audio Device.

**To Reproduce:**

Slide the knob of the Z407

**Screenshots:**

**OS Version:**
 - Windows 10 Version: 
Windows 11 DEV Build 22000.160
**ModernFlyouts Version:**
 - Version: 
v0.9.3.0
**Additional context:**

**Comment by Viniuau** [comment]
Also happens to me with my Kotion Each G2200 headset. I usually tweak the volume using the buttons on my headset.
The volume is stuck at 100 and doesn't change.

**Comment by adx74** [comment]
> Also happens to me with my Kotion Each G2200 headset. I usually tweak the volume using the buttons on my headset.
> The volume is stuck at 100 and doesn't change.

Thats exactly the same for me, i think device sound control doesn't work in ModernFlyouts @Viniuau

**Comment by Viniuau** [comment]
I managed to get it working now by changing the current device using EarTrumpet. By any chance, do you use Voicemeeter? Because I do and I have it selected as the output all the time, but just now I changed it to my headphones and it worked. The natural behavior would be to change the current output device being affected.

Headphones -> Change volume when VoiceMeeter Input is set as the root -> Shows a fixed amount
Headphones -> Change volume when headphones are selected on EarTrumpet -> Shows the correct amount and changes accordingly.

**Comment by Cyberdroid1** [comment]
@Viniuau you are changing the default audio device with Eartrumpet, which is why it shows the flyout.

**Comment by Viniuau** [comment]
Yeah, it's just that the native Windows flyout changes it automatically when pressed.
Is it something difficult to replicate/implement as a feature?
